client/generator-carousel-admin/js/main.js

```diff
@@ -4,6 +4,44 @@ var default_url      = 'https://en.wikipedia.org/wiki/Financial_Times';
 var default_duration = 10;
 var default_url = "https://en.wikipedia.org/wiki/Static_web_page";
 
+var keyUpTimeout = null,
+	table = undefined;
+
+function appendNewInputToForm(where){
```

Comment:
That's an oversight, meant to have it so that any element could have it an input appended to it, but it seemed like overkill given the general specificity of the function.